ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -505,6 +367,172 @@ const triggerSet: TriggerSet<Data> = {
+            `R1S Leaping clone collector: Missing ActorSetPos line for actor ID ${matches.sourceId}`,
+          );
+          return;
+        }
+        const cloneNorthSouth = parseFloat(actorSetPosEntry.y) < 100 ? 'north' : 'south';
+
+        if (data.storedLeaps.oneTwoPaw.firstCleaveSide !== undefined) {
+          if (data.storedLeaps.oneTwoPaw.northSouth === undefined) {
+            data.storedLeaps.oneTwoPaw.northSouth = cloneNorthSouth;
+            return;
+          }
+        }
+
+        if (data.storedLeaps.quadCross.leftRight !== undefined) {
+          if (data.storedLeaps.quadCross.northSouth === undefined) {
+            data.storedLeaps.quadCross.northSouth = cloneNorthSouth;
+            return;
+          }
+        }
+
+        if (
+          data.storedLeaps.oneTwoPaw.northSouth !== undefined &&
+          data.storedLeaps.quadCross.northSouth !== undefined
+        ) {
+          if (
+            data.storedLeaps.oneTwoPaw.resolved !== true &&
+            data.storedLeaps.oneTwoPaw.northSouth === cloneNorthSouth
+          ) {
+            data.storedLeaps.oneTwoPaw.resolved = true;
+            let dir: 'dirE' | 'dirW';
+
+            if (data.storedLeaps.oneTwoPaw.northSouth === 'north') {
+              if (data.storedLeaps.oneTwoPaw.leftRight === 'left')
+                dir = 'dirE';
+              else
+                dir = 'dirW';
+            } else {
+              if (data.storedLeaps.oneTwoPaw.leftRight === 'left')
+                dir = 'dirW';
+              else
+                dir = 'dirE';
+            }
+
+            let inOut: 'in' | 'out' = 'in';
+            if (data.storedLeaps.oneTwoPaw.leftRight !== data.storedLeaps.oneTwoPaw.firstCleaveSide)
+              inOut = 'out';
+
+            return output.healerStacks!({ dir: output[dir]!(), inOut: output[inOut]!() });
+          }
+
+          if (
+            data.storedLeaps.quadCross.resolved !== true &&
+            data.storedLeaps.quadCross.northSouth === cloneNorthSouth
+          ) {
+            data.storedLeaps.quadCross.resolved = true;
+            let dir: 'dirE' | 'dirW';
+
+            if (data.storedLeaps.oneTwoPaw.northSouth === 'north') {
+              if (data.storedLeaps.oneTwoPaw.leftRight === 'left')
+                dir = 'dirE';
+              else
+                dir = 'dirW';
+            } else {
+              if (data.storedLeaps.oneTwoPaw.leftRight === 'left')
+                dir = 'dirW';
+              else
+                dir = 'dirE';
+            }
+            return output.proximity!({ dir: output[dir]!() });
+          }
+        }
+      },
+
+      outputStrings: {
+        dirE: Outputs.dirE,
+        dirW: Outputs.dirW,
+        in: {
+          en: 'In => Out',
+        },
+        out: {
+          en: 'Out => In',
+        },
+        healerStacks: {
+          en: 'Go ${dir}, ${inOut}, Healer Stacks',
+        },
+        proximity: {
+          en: 'Go ${dir}, Proximity Baits/Spreads',
```

Comment:
```suggestion
        in: {
          en: 'In + Healer Stacks => Out',
        },
        out: {
          en: 'Out + Healer Stacks => In',
        },
        healerStacks: {
          en: 'Go ${dir} => ${inOut}',
        },
        proximity: {
          en: 'Go ${dir} => Proximity Baits + Spreads',
        },
```

Purely stylistic, feel free to disregard -- there's no established convention, at least not one that's consistently followed, but the predominant one I've seen used (which makes sense to me at least) is to use `=>` to indicate a sequence of steps/mechanics, `/` to indicate an either/or mechanic, and `+` to indicate mechanics that are resolved simultaneously.

Reply:
Yea, that tends to be a good strategy